backpack-lint/src/main/java/net/skyscanner/backpack/lint/check/BpkComposeComponentUsageDetector.kt

```diff
@@ -100,6 +105,11 @@ class BpkComposeComponentUsageDetector : Detector(), SourceCodeScanner {
       webName = "text",
       componentsToReplace = setOf("androidx.compose.material.Text")
     ),
+    TRI_STATE_CHECKBOX(
+      fullName = "net.skyscanner.backpack.compose.checkbox.BpkCheckbox",
+      webName = "checkbox",
+      componentsToReplace = setOf("androidx.compose.material.TriStateCheckbox")
```

Comment:
you can also provide multiple package names in `CHECKBOX`, rather than creating another enum here (similar to buttons)